argus/recipes/cloud/windows.py

```diff
@@ -831,3 +831,21 @@ def prepare_cbinit_config(self, service_type):
         self._cbinit_conf.set_conf_value(
             name="user_password_length",
             value="3")
+
+
+class CloudbaseInitNoopRecipe(CloudbaseinitRecipe):
+    def create_mock_metadata(self, service_type):
+        """Create the mocked metadata."""
+        pass
+
+    def delete_mock_metadata(self):
+        """Delete the mocked metadata."""
+        pass
+
+    def prepare_cbinit_config(self, service_type):
```

Comment:
take a look at other recipes and you can inherit one of them that implements the preparation as you desire